src/hooks/useGetSightingsByLocation.ts

```diff
@@ -0,0 +1,48 @@
+import { useState } from 'react';
+import { API_PATH } from '@/utils/constants';
+import { DeerSighting } from '@/interfaces/DeerSighting';
+
+const useGetSightingsByLocation = () => {
+  const [sightings, setSightings] = useState<DeerSighting[] | null>(null);
+  const [error, setError] = useState<string | null>(null);
+  const [loading, setLoading] = useState<boolean>(false);
```

Comment:
delete, when you use `useQuery` or `useMutation`, those functions are already built in with error and loading variables, so these will end up being unused.

Reply:
also delete any code related to the error and loading useStates